maps/site404/site404shuttles.dm

```diff
@@ -0,0 +1,98 @@
+
+//SCP-106 Access Lift
+
+/datum/shuttle/autodock/ferry/lift106
+	name = "SCP-106 Lift" //You put the name in the shuttle_tag on the shuttle control console.
+	sound_takeoff = 'sounds/effects/lift_heavy_start.ogg'
+	warmup_time = 5
+	shuttle_area = list(/area/site404/shuttle/lift106)
+	waypoint_station = "nav_lift106_start"
+	waypoint_offsite = "nav_lift106_out"
+
+/obj/effect/shuttle_landmark/lift106/start
+	name = "SCP-106 Lower Access Lift"
+	landmark_tag = "nav_lift106_start"
+	base_turf = /turf/simulated/floor/plating
+	base_area = /area/site404/containmentchambers/scp106/entrance
+
+/obj/effect/shuttle_landmark/lift106/out
+	name = "SCP-106 Upper Access Lift"
+	landmark_tag = "nav_lift106_out"
+	base_turf = /turf/simulated/open
+	base_area = /area/site404/containmentchambers/scp106/entrance
+
+//SCP-106 Containment Unit Shuttle
```

Comment:
@checkraisefold An example usage of the shuttle audio PR in https://github.com/Foundation-19/Foundation-19/pull/1802. Without it this takeoff will only be audible if you're on the same Z that the "shuttle" is leaving from, and if not it will be complete silence then a giant structure will appear. Audio is essential for giving these objects weight.